Sources/Routing/Parameterizable.swift

```diff
@@ -0,0 +1,25 @@
+public protocol Parameterizable {
+    /// the unique key to use as a slug in route building
+    static var uniqueSlug: String { get }
```

Comment:
Having a static variable here, is it possible to have the same type more than once, ie: `drop.get(String, "foo", String)`